[BUG]: Removal of readme.md, getting_started.md from docs also removed 'docs/develop' directory

### Describe the issue:

make read-docs ->

...
cp README.md docs/develop/getting_started.md
cp: reguläre Datei 'docs/develop/getting_started.md' kann nicht angelegt werden: Datei oder Verzeichnis nicht gefunden
make: *** [Makefile:54: docs] Fehler 1


### Reproduceable code example:

```python
make read-docs
```


### Error message:

```shell
cp README.md docs/develop/getting_started.md
cp: reguläre Datei 'docs/develop/getting_started.md' kann nicht angelegt werden: Datei oder Verzeichnis nicht gefunden
make: *** [Makefile:54: docs] Fehler 1
```


### Version information:

5e2eae7331ee39dfe2ce323214126a9fad8a8000

--- a/Makefile
+++ b/Makefile
@@ -51,6 +51,7 @@ dist: pip
 
 # Target to generate HTML documentation
 docs: pip-dev
+	mkdir -p docs/develop
 	cp README.md docs/develop/getting_started.md
 	# remove top level header and coresponding description
 	sed -i '/^##[^#]/,$$!d' docs/develop/getting_started.md

--- a/docs/akkudoktoreos/serverapi.rst
+++ b/docs/akkudoktoreos/serverapi.rst
@@ -7,7 +7,7 @@
 Server API
 ##########
 
-For a more detailed documentation see the Swagger interface: `EOS OpenAPI Spec <https://petstore3.swagger.io/?url=https://raw.githubusercontent.com/Akkudoktor-EOS/EOS/refs/heads/main/openapi.json>`_
+For a more detailed documentation see the Swagger interface: `EOS OpenAPI Spec <https://petstore3.swagger.io/?url=https://raw.githubusercontent.com/Akkudoktor-EOS/EOS/refs/heads/main/docs/akkudoktoreos/openapi.json>`_
 
 .. openapi:: openapi.json
    :examples:

